app/helpers/question_view_helper.rb

```diff
@@ -13,6 +13,26 @@ def legend(question, page_header, conditional)
     }
   end
 
+  def answer_option_synonyms(resource)
+    return unless resource.instance_of?(Scheme)
```

Comment:
I find `resource.is_a? Scheme` slightly more readable, but I'm pretty sure they are aliases, I jsut mention it in case you know of one and not the other